base/ltclass.dtx

```diff
@@ -1072,7 +1072,7 @@
 %<latexrelease>     \csname ver@\@currname.\@currext\endcsname{#1}%
 %<latexrelease>\ifx\@currext\@clsextension
 %<latexrelease>    \typeout{Document Class: \@gtempa\space#1}%
-%<latexrelease>  \else
+%<latexrelease>  \else@xprocess@ptions
```

Comment:
Is this on purpose?

Reply:
Thanks! The good news is that the test suite caught this, I saw the failure last night but it was too late to trace code so I left it till this morning, but power of the internet: you point out the bad line before I wake up...